osbuild/buildroot.py

```diff
@@ -13,10 +13,16 @@
 
 
 class BuildRoot:
-    def __init__(self, root, path="/run/osbuild"):
+    def __init__(self, root, path="/run/osbuild", var_path="/var/tmp"):
         self.root = tempfile.mkdtemp(prefix="osbuild-buildroot-", dir=path)
         self.api = tempfile.mkdtemp(prefix="osbuild-api-", dir=path)
+        self.var = tempfile.mkdtemp(prefix="osbuild-var-", dir=var_path)
```

Comment:
We usually hard-code paths like `/var/tmp`. Do you mind just dropping `var_path`?

At least, I don't see a reason for anyone to use something else.

Reply:
I don't mind, i've pushed the change. :)